build_tools/test_wheels.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# test that we are able to load pyrfr
+pytest -v /project/tests/pyrfr_unit_test_binary_regression_forest_transformed_data.py
```

Comment:
Hey, would it be possible to run all unit tests for python or would that be too expensive?